src/views/Datasets.vue

```diff
@@ -47,20 +61,24 @@
 					<small>{{ preferredLanguage(row.item.description) }}</small>
 				</p>
 			</template>
-			<template slot="actions" slot-scope="row">
-				<div class="actions">
-					<b-button variant="primary" size="sm" @click.stop="open(row.item.id)"><font-awesome-icon icon="pen" fixed-width />Edit</b-button>
-					<b-button variant="primary" size="sm" @click.stop="view(row.item.identifier)" :disabled="row.item.identifier == null"><font-awesome-icon icon="external-link-alt" fixed-width />View in Etsin</b-button>
-
-					<b-dropdown variant="primary" right text="More" size="sm">
-						<b-dropdown-item-button size="sm" v-b-modal="'dataset-versions-modal'" @click="activeInModal = row.item.id" :disabled="row.item.versions < 1">
+			<template slot="row-details" slot-scope="row">
+				<b-card>
+					<b-button-group class="row-card-primary-actions">
+						<b-button size="sm" variant="primary" @click.stop="view(row.item.identifier)" :disabled="row.item.identifier == null"><font-awesome-icon icon="external-link-alt" fixed-width />View in Etsin</b-button>
```

Comment:
Is there some hidden character in front of this line?

Reply:
I will check

Reply:
should be fixed